qa-params.json

```diff
@@ -1,5 +1,6 @@
 {
   "Parameters": {
+    "TESTMODE": 0,
```

Comment:
In order to end up in a lambdas environment variables, the parameter has also to be provided in the template.yaml file for each service.

Reply:
Yes, that's right!